README.md

```diff
@@ -41,22 +159,7 @@ FROM gcr.io/distroless/base-debian10
 COPY --from=conda /opt/env /opt/env
 ```
 
-## installation
-
-```
-pip install conda-lock
-```
-
-## usage
-
-```bash
-# generate the lockfiles
-conda-lock -f environment.yml -p osx-64 -p linux-64
-
-# create an environment from the lockfile
-conda-lock install [-p {prefix}|-n {name}] conda-linux-64.lock
-
-# alternatively, use conda command directly
-conda create -n my-locked-env --file conda-linux-64.lock
-```
-
+[conda]: https://docs.conda.io/projects/conda
+[metayaml]: https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html
+[mapping]: https://github.com/regro/cf-graph-countyfair/blob/master/mappings/pypi/grayskull_pypi_mapping.yaml
```

Comment:
I just realized that this list has tons of Caps diff that are not really naming diffs. It makes it hard to read and find real differences :-/

Anyway, not a problem for conda-lock/

Reply:
that list is just link references for reference style markdown links, so not human visible